### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by taehoonlee**
This PR fixes some typos: `defaut`, `overriden`, and `iteratble`.

### CODE REVIEW

File: lib/matplotlib/tests/test_table.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def test_customcell():
 def test_auto_column():
     fig = plt.figure()
 
-    #iteratble list input
+    #iterable list input
```
Comment (QuLogic) [inline lib/matplotlib/tests/test_table.py]: Add space after `#`.

File: lib/matplotlib/tests/test_table.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def test_auto_column():
     tb1.set_fontsize(12)
     tb1.auto_set_column_width([-1, 0, 1])
 
-    #iteratble tuple input
+    #iterable tuple input
```
Comment (QuLogic) [inline lib/matplotlib/tests/test_table.py]: Add space after `#`.

File: lib/matplotlib/tests/test_table.py (hunk 1 above)
Comment (taehoonlee) [inline lib/matplotlib/tests/test_table.py]: Updated.

File: lib/matplotlib/tests/test_table.py (hunk 2 above)
Comment (taehoonlee) [inline lib/matplotlib/tests/test_table.py]: Updated.

File: lib/matplotlib/backends/web_backend/jquery/js/jquery-ui.js [hunk 3]
```diff
@@ -1034,7 +1034,7 @@ var mouse = $.widget("ui.mouse", {
 		return this.mouseDelayMet;
 	},
 
-	// These are placeholder methods, to be overriden by extending plugin
+	// These are placeholder methods, to be overridden by extending plugin
```
Comment (QuLogic) [inline lib/matplotlib/backends/web_backend/jquery/js/jquery-ui.js]: Probably shouldn't change bundled JQuery bits.

Comment (taehoonlee) [inline lib/matplotlib/backends/web_backend/jquery/js/jquery-ui.js]: Updated.

**Pull request merged**